workspaces/shims-0.19/src/Elm19Compatible/Time/Extra.elm

```diff
@@ -0,0 +1,14 @@
+module Elm19Compatible.Time.Extra exposing (
+    toIsoString
+    , floor
+    , diff
+    , compare
+    )
+
+import Time.Extra
+
+
+floor = Date.Extra.floor
+diff = Date.Extra.diff
+compare = Date.Extra.compare
+toIsoString = Date.Extra.toIsoString
```

Comment:
Is there supposed to be a `shims-0.18` version of this?

Reply:
Actually, this could isn't used, isn't tested and doesn't even compile (Date isn't imported). It must have been some WIP thing that accidentally got included. I'll just remove it.